src/bucket/BucketList.cpp

```diff
@@ -111,8 +111,9 @@ BucketLevel::commit()
     if (mNextCurr.isLive())
     {
         setCurr(mNextCurr.resolve());
-        // CLOG(DEBUG, "Bucket") << "level " << mLevel << " set mCurr to "
-        //            << mCurr->getEntries().size() << " elements";
+        // CLOG_DEBUG(Bucket, "level {}{}{} elements", mLevel, " set
```

Comment:
looks like your script didn't work here. I'd just remove it or convert it to a TRACE level log

Reply:
you can look for `// CLOG` and `// LOG` to find all those places

Reply:
fixed.